cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -840,18 +851,23 @@ instance (Show txid, Show tx)
 
 instance Show peerAddr => ToObject (ConnectionId peerAddr) where
     toObject _verb ConnectionId { localAddress, remoteAddress } =
-      mkObject [ "localAddress"  .= show localAddress
+      mkObject [ "kind"          .= String "ConnectionId"
+               , "localAddress"  .= show localAddress
                , "remoteAddress" .= show remoteAddress
                ]
 
+instance Show peerAddr => ToJSON (ConnectionId peerAddr) where
+  toJSON = Object . toObject NormalVerbosity
+
```

Comment:
This instance looks odd to me.  I think `ToObject` instance should be build in terms for `ToJSON` instance.  I would go even further an remove the `ToObject` instance of `ConnectionId` and use `ToJSON` instead.

Reply:
That sound way better! Thanks 😄